src/app_ui.py

```diff
@@ -142,15 +142,21 @@ def create_new(self):
                     reference_type = "phdthesis"
                 case "6":
                     return
-                case default:  # pylint: disable=unused-variable
+                case _:
                     self.root.io_handler.write_output("Please pick a valid option")
                     continue
 
             for field in required_fields.split("|"):
-                fields_data[field] = self.root.io_handler.read_input(
-                    f"insert {field}: "
-                )
-
+                required = {"author", "title"}
+                if field in required:
+                    value = ""
+                    while value == "":
+                        value = self.root.io_handler.read_input(f"insert {field}: ")
+                    fields_data[field] = value
+                else:
+                    fields_data[field] = self.root.io_handler.read_input(
+                        f"insert {field}: "
+                    )
```

Comment:
This forces the user to actually fill _author_ and _title_ fields. Previously they could be left empty, which crashes the program.